src/App.js

```diff
@@ -12,12 +12,62 @@ import JournalForm from './Journal/JournalForm'
 import JournalList from './Journal/JournalList'
 import { useState } from 'react'
 import { loadLocally, saveLocally } from './Journal/util'
+import Header from './common/Header'
 
 export default function App() {
   const [journalEntries, setJournalEntries] = useState(
     loadLocally('journalEntries') ?? []
   )
 
+  useEffect(() => saveLocally('journalEntries', journalEntries), [
+    journalEntries,
+  ])
+
+  return (
+    <>
+      <Switch>
+        <Route exact path="/"></Route>
+        <Route path="/">
+          <Header />
+        </Route>
```

Comment:
Just curios, does this make your header appear on every page execpt your homepage? Otherwise I would not understand the two path's to "/"

Reply:
Yes, exactly. I couldn't find another way to put the header on every page except for the homepage.